packages/template-retail-react-app/app/pages/account/index.jsx

```diff
@@ -96,10 +96,10 @@ const Account = () => {
     // If we have customer data and they are not registered, push to login page
     // Using Redirect allows us to store the directed page to location
     // so we can direct users back after they are successfully log in
-    if (customer.authType != null && !customer.isRegistered) {
-        const path = buildUrl('/login')
-        return <Redirect to={{pathname: path, state: {directedFrom: location.pathname}}} />
-    }
+    // if (customer.authType != null && !customer.isRegistered) {
+    //     const path = buildUrl('/login')
+    //     return <Redirect to={{pathname: path, state: {directedFrom: location.pathname}}} />
+    // }
```

Comment:
The account page seemed to be in a non-working state when I logged in. I assume that you intention is to leave is in this state until we get to that ticket?

Reply:
that's right, i added a comment to clarify.